.github/workflows/drush-security-check.yml

```diff
@@ -0,0 +1,28 @@
+name: Drush Security Check
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  check:
+    name: PHP ${{ matrix.php-versions }}
```

Comment:
Can we change name:
`name: Drush Security Check for PHP ${{ matrix.php-versions }}`
@vipin-mittal-acquia @vishalkhode1

Reply:
@rajeshreeputra : Please refer to the attached [screenshot](https://user-images.githubusercontent.com/109503136/211245454-26fd672b-bb25-4158-866f-44e73921853c.png) to see what you mean by the changes.

Reply:
@rajeshreeputra: Made changes accordingly